src/collections/_documentation/workflow/search.md

```diff
@@ -6,33 +6,37 @@ sidebar_order: 3
 Search is available on several major Sentry views: Issues, Events, and Releases.
 
 {% capture __alert_content -%}
-Discover is Sentry's query builder for aggregating raw event data and has its own unique syntax not covered here. For more information, see full Discover documentation.
+Discover is Sentry's powerful query builder for aggregating raw event data and has its own unique syntax not covered here. For more information, see [full Discover documentation]({%- link _documentation/workflow/discover.md -%}).
 {%- endcapture -%}
 {%- include components/alert.html
   title="Note"
   content=__alert_content
+  level="info"
 %}
 
+&nbsp;
 ## Syntax
 
-Queries are constructed using a `token:value` pattern:
+Queries are constructed using a `token:value` pattern, with an optional raw search at the end:
```

Comment:
It could make sense to describe this as a `key:value` pattern, and then each `key:value` element is described as a `token`? Then raw search is also an optional token included at the end.

Reply:
Nice! Yeah. I like that. :)